curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-the-date-object-by-building-a-date-formatter/6538935e2ab721beedb137c4.md

```diff
@@ -1,19 +1,19 @@
 ---
 id: 6538935e2ab721beedb137c4
-title: Step 25
+title: Step 23
 challengeType: 0
-dashedName: step-25
+dashedName: step-23
 ---
 
 # --description--
 
-Finally, join the results from the array reversal. Use a `"-"` as the separator. Remember to use method chaining.
+Like in the previous step, use method chaining to `split`, `reverse`, and `join` the array.
```

Comment:
```suggestion
Like in the previous step, use method chaining to `split`, `reverse`, and `join` the array. Use `'-'` in the split and join methods.
```

I think we should add a mention here to use the `-` on the split and join.

Reply:
Thank you very much! Would it be okay if I look into this next week? I will let you know if I need any help.